packages/design-system/src/components/Input/Input.tsx

```
@@ -0,0 +1,149 @@
+  const currentType =
+    isPasswordInput && isPasswordVisible ? "text" : type;
+
+  return (
+    <div className={cn("w-full", className)}>
+      <div
+        className={cn(
+          "flex w-full items-center transition-colors",
+          isAddressInput
+            ? "rounded-[999px] bg-gray-200"
+            : [
+                "rounded-[8px]",
```

ESLint

[error] Expected indentation of 14 spaces but found 16.

(indent)